app/account/page.tsx

```diff
@@ -0,0 +1,143 @@
+'use client'
+
+import {
+  Heading,
+  Flex,
+  Box,
+  Container,
+  VStack,
+  Image,
+  Text,
+  useColorModeValue,
+  Spacer
+} from '@chakra-ui/react'
+import { CiEdit } from 'react-icons/ci'
+import { FaRegTrashAlt } from 'react-icons/fa'
+import { MdAccountCircle } from 'react-icons/md'
+import { PrimaryButton, AlertButton } from '@/components/button'
+import { Link } from '@/components/link'
+import { Loading } from '@/components/loading'
+import { useGetUserQuery } from '@generated/api'
+
+export default function AccountPage() {
+  const bg = useColorModeValue('white', 'gray.800')
+  const color = useColorModeValue('black', 'gray.300')
+  // TODO: Remove this line after implementing login
+  const tmpUuid = 'a1f4509d-39f6-4de7-921d-201ec3d4578f'
+  const { data, loading, error } = useGetUserQuery({
+    variables: { uuid: tmpUuid }
+  })
+  const user = data?.usersCollection?.edges[0].node
+
+  // TODO: Refactoring error handling using Apollo ErrorLink
+  //       https://www.apollographql.com/docs/react/api/link/apollo-link-error/
+  if (error) {
+    // NOTE: If error is thrown, Next.js will render automatically account/error.tsx
+    throw new Error(error.message)
+  }
+
+  return (
+    <>
+      {loading || !data?.usersCollection ? (
+        <Loading />
+      ) : (
+        <Box
+          as="main"
+          minH={{ base: '86vh', lg: '78vh' }}
+          bg={bg}
+          color={color}
+        >
+          <Container
+            minW={{ base: '100%', lg: 'container.sm' }}
+            display="flex"
+            flexDirection="column"
+            alignItems={{ base: 'space-around', lg: 'flex-start' }}
+            gap={{ base: '40px', lg: '60px' }}
+            mt={{ base: '20px', md: '40px' }}
```

Comment:
You might have to add `mb={{ base: '60px', md: '70px' }}` to the container.  
Right now, the delete button is touching the baseline on desktop size 
<img width="200" alt="Screenshot 2023-12-22 at 11 58 14" src="https://github.com/tabi-memo/frontend/assets/70562492/b80f18f0-54a2-436f-82ba-964cdac2b404">

Reply:
Fixed it.
https://github.com/tabi-memo/frontend/pull/13/commits/326c111e6dddf1039531d53b93446d715d48ea12